neXtProt.pm

```diff
@@ -0,0 +1,347 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+    
+=cut
+
+=head1 NAME
+
+ neXtProt
+
+=head1 SYNOPSIS
+
+ mv neXtProt.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin neXtProt
+ ./vep -i variations.vcf --plugin neXtProt,max_set=1
+
+=head1 DESCRIPTION
+
+ This is a plugin for the Ensembl Variant Effect Predictor (VEP) that
+ retrives data for missense variants from neXtProt, which is a comprehensive 
+ human-centric discovery platform that offers integration of and navigation 
+ through protein-related data (https://www.nextprot.org/).
+
+ Please cite the neXtProt publication alongside the VEP if you use this resource:
+ https://doi.org/10.1093/nar/gkz995
+ 
```

Comment:
We need a comment - only suitable for small sets of variants as an additional individual remote API query run for each variant.